.github/workflows/ci.yml

```diff
@@ -9,8 +9,8 @@ on:
       - 'LICENSE'
       - '.gitignore'
 env:
-  node_version: "12.x"
-  tf_version: "0.14.3" # must match value in terraform-iac/*/app/main.tf
+  node_version: "14.x"
+  tf_version: "0.14.7" # must match value in terraform-iac/*/app/main.tf
```

Comment:
nit: Why not 1.0.0?

Reply:
Because I was basically doing a diff against files from hw-fargate-api and wanted to keep this already-large commit as focused as possible.

We should definitely update both to TF 1.0.0.